contributor-docs/release-guide.md

```diff
@@ -231,9 +231,12 @@ Configure access to the [Apache Nexus repository](https://repository.apache.org/
 
 #### Update base image dependencies for Python container images
 
-Tracked in Github issue https://github.com/apache/beam/issues/27944
+The Python base container images have static pinned `requirements.txt` that are
+designed to be compatible with our dependency constraints but also not cause
+runtime installs to occur, which slow things down immensely.
```

Comment:
```suggestion
runtime installs to occur, which slow things down immensely.
These need to be updated at least once per release cycle to avoid
out of date dependencies.
```

This doesn't actually describe the task to be performed (I know its in the title, but I think a little extra clarification would help)